py_trees_ros/utilities.py

```diff
@@ -161,10 +161,10 @@ def qos_profile_latched():
     Convenience retrieval for a latched topic (publisher / subscriber)
     """
     return rclpy.qos.QoSProfile(
-        history=rclpy.qos.QoSHistoryPolicy.RMW_QOS_POLICY_HISTORY_KEEP_LAST,
+        history=rclpy.qos.QoSHistoryPolicy.KEEP_LAST,
         depth=1,
-        durability=rclpy.qos.QoSDurabilityPolicy.RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL,
-        reliability=rclpy.qos.QoSReliabilityPolicy.RMW_QOS_POLICY_RELIABILITY_RELIABLE
+        durability=rclpy.qos.QoSDurabilityPolicy.TRANSIENT_LOCAL,
+        reliability=rclpy.qos.QoSReliabilityPolicy.RELIABLE
```

Comment:
@stonier These changes just remove deprecation warnings but aren't necessary... just nice to have.

Reply:
Same deal, let's handle this separately as it's a breaking change across distros.

Reply:
Resolved in https://github.com/splintered-reality/py_trees_ros/pull/187.